tools/reviewer/service/tests/CodeReviewServiceTest.java

```diff
@@ -59,9 +60,9 @@
   private static final String TEST_FILE = "test_file.txt";
   private static final String TEST_FILE_CONTENTS = "Some test file contents\n";
   private static final String TEST_WORKSPACE = "ws1";
+  private static final String COMMIT_MESSAGE = "Some commit message";
 
   private GitRepoFactory gitRepoFactory;
-  private GitRepo gitRepo;
   private String aaBaseFolder;
```

Comment:
- TestComponent component;
- Server server;
- ManagedChannel channel;
- CodeReviewServiceGrpc.CodeReviewServiceBlockingStub blockingStub;

Can we do private access?